nyc_taxi/dashboard.ipynb

```diff
@@ -33,19 +33,34 @@
     "from holoviews.operation.datashader import rasterize, dynspread\n",
     "from holoviews.element.tiles import EsriStreet\n",
     "\n",
-    "hv.extension('bokeh', logo=False)\n",
-    "\n",
-    "usecols = ['dropoff_x','dropoff_y','pickup_x','pickup_y','dropoff_hour','pickup_hour','passenger_count']\n",
-    "df = dd.read_parquet('data/nyc_taxi_wide.parq')[usecols].persist()\n",
-    "opts = dict(width=1000,height=600,xaxis=None,yaxis=None,bgcolor='black',show_grid=False)\n",
-    "cmaps = ['fire','bgy','bgyw','bmy','gray','kbc']\n",
-    "\n",
+    "hv.extension('bokeh', logo=False)"
```

Comment:
No idea why `logo=False` was set, I don't think the logo hurts at all, let's keep it.

```suggestion
    "hv.extension('bokeh')"
```